modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectDefinitionLocalServiceTest.java

```diff
@@ -181,6 +196,22 @@ public void testAddCustomObjectDefinition() throws Exception {
 			"Plural label is null for locale " + LocaleUtil.US.getDisplayName(),
 			() -> _addCustomObjectDefinition("Test", "Test", ""));
 
+		// Object folder does not exist
```

Comment:
It'd be better if this assetFailure came first than the `Plural labe is null`.
Do you agree?

Reply:
Agreed.